src/overlays/actors/ovl_Bg_Ikana_Dharma/z_bg_ikana_dharma.h

```diff
@@ -3,15 +3,20 @@
 
 #include "global.h"
 
+#define BGIKANADHARMA_GET_SWITCHFLAG(thisx) (((thisx)->params >> 8) & 0x7F)
+#define BGIKANADHARMA_IS_CHILD(thisx) (((thisx)->params >> 5) & 1)
+#define BGIKANADHARMA_NUM_SEGMENTS(thisx) ((thisx)->params & 0xF)
+#define BGIKANADHARMA_PARAM(switchFlag, isChild, numSegments) (((switchFlag & 0x7F) << 8) | ((isChild & 1) << 5) | (numSegments & 0xF))
```

Comment:
```suggestion
#define BGIKANADHARMA_PARAMS(switchFlag, isChild, numSegments) (((switchFlag & 0x7F) << 8) | ((isChild & 1) << 5) | (numSegments & 0xF))
```
Small nit, but `PARAMS` would probably be better to match the struct member name.